### ISSUE TITLE
FGR with rotation data given as an input

### ISSUE THREAD

**Issue by herbauau** [intel-isl/Open3D#787]
As part of a school project, I need to create an odometry algorithm to determine the distances of a vehicle. This algorith would have in input two point clouds: one source and one target, but also INS data, that is the rotations of the vehicle.

For the moment, I use the FGR method implemented in Python and using open3d, but the output of this algorithm is a transformation matrix where translations AND rotations are calculated. I would like to modify the "FastGlobalRegistration" or "FastGlobalRegistrationOption" function to make it take into account INS data and calculate translations from the source point cloud to the target point cloud.

From now, I have inversed, in another code, the matrix obtained via this algorithm to determine translations from the INS rotations. This solution, however, is not precise enough and takes more time than a single algorithm taking into account the rotations.

This project aims to determine the position of a vehicle without GPS, that is with an initial known position, rotations given by INS data and translations calculated via this algorithm. It must be finished before March, and I have a few knowledge in programming (I only know Python and a few Matlab).

Thanks in advance,

**Comment by syncle** [comment]
Hi @herbauau. If the registration task is not global registration (you mentioned odometry), that would be better to use ICP. Translation only registration is related to #409. I will work for this when I am available.

**Comment by germanros1987** [comment]
Issue closed due to lack of activity. Feel free to open a new issue if you still consider this is a critical matter.
